src/components/molecules/HamburgerMenu/HamburgerMenu.tsx

```diff
@@ -45,30 +50,30 @@ const MenuItemLink = ({ text, link }: { text: string; link: string }) => {
     );
 };
 
-const MobileItemLink = ({ text, href }: { text: string; href: string }) => {
+const MobileItemLink = ({
+    text,
+    href,
+    target,
+}: {
+    text: string;
+    href: string;
+    target?: string;
+}) => {
+    const router = useRouter();
+    const isActive = router.pathname === href;
     return (
         <Menu.Item>
-            {({ active }) => (
```

Comment:
No longer need active state from Menu. Only styles are:

1) Default
2) Underline to indicate that user is in the associated page (underline `Markets` for /dashboard)